flang/include/flang/Lower/Runtime.h

```diff
@@ -91,6 +91,11 @@ void genTransferSize(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,
                      mlir::Value resultBox, mlir::Value sourceBox,
                      mlir::Value moldBox, mlir::Value size);
 
+/// generate runtime call to transfer intrinsic
+void genTransfer(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,
+                 mlir::Value resultBox, mlir::Value sourceBox);
```

Comment:
Was this `genTransfer` here by accident?

Reply:
Yeah, that was accidental. Removed and updated comment. Thanks for pointing out.